tests/doctestUtils.h

```diff
@@ -0,0 +1,21 @@
+#include <algorithm>
```

Comment:
This header is missing `#pragma once`. Could that be related to the linker error @weegeekps was seeing: https://github.com/CesiumGS/cesium-omniverse/pull/346#issuecomment-1593673073

Reply:
I'd be surprised if it was related but I've just pushed a fix. Thanks for the catch!